gpflow/kernels/stationaries.py

```diff
@@ -115,6 +115,12 @@ class AnisotropicStationary(Stationary):
     input dimension.
     """
 
+    def __init__(self, *args, **kwargs):
```

Comment:
Best to copy the base class `__init__` parameters for better docstrings and autocomplete.

Reply:
Done!